experiments/pages/lightningModalLauncher/README.md

```diff
@@ -0,0 +1,24 @@
+# Lightning Modal 
+
+## Description:
+This page contains a skuid page with a button and a lightning component. The skuid page publishes an event that the lightning component listens for. When the event is published a lightning component is called and it launches a lightning modal where you could put just about anything from lightning.
+
+## Note:
+It's important to note if you are trying to modify or access data then you will have to that that the lightning modal takes in an object and not a string so you will have to modify `auraEvt` var in `lightningModalLauncher.xml`  
```

Comment:
Suggesting re-write would be:
"It's important to note that in this example the lightning component is listening for an object to be passed in, (rather than a string), which is why we are using JS to publish an event rather than the declarative "Publish Event" action. If you need to modify the parameters being passed into the modal/flow it will need to be modified in the Javascript and the lightning component.